openrtb2/bid_request.go

```diff
@@ -90,7 +90,7 @@ type BidRequest struct {
 	// Description:
 	//    Indicator of test mode in which auctions are not billable,
 	//    where 0 = live mode, 1 = test mode.
-	Test int8 `json:"test,omitempty"`
```

Comment:
should we add some kind of comment saying this behaviour is Seedtag's custom?

Reply:
added a document explaining changes done and workflow